build.cake

```diff
@@ -294,7 +294,11 @@ Task("PackageNugetPackages")
             var settings = new DotNetCorePackSettings
             {
                 Configuration = configuration,
-                OutputDirectory = artifactsPackagesDirectory
+                OutputDirectory = artifactsPackagesDirectory,
```

Comment:
I don't really have the context to review this.  Can you describe why each of these is necessary so everyone understands?

Reply:
Robert can confirm, but I believe we have `NoBuild` since we've already built in a previous step, `IncludeSymbols` to allow for debugging/line numbers in stack traces, and the `MSBuildSettings` w/ "PackageVersion" fixes the `0.0.0.0` problem.